src/pages/Home/Home.tsx

```diff
@@ -1,12 +1,12 @@
 import { Button } from "@/components/ui/button";
-import { ApplicationRoutes } from "@/enums/ApplicationRoutesEnums";
+import { RotasAplicacao } from "@/enums/RotasAplicacaoEnums";
 import { useNavigate } from "react-router-dom";
 
 const Home = () => {
   const navigate = useNavigate();
 
-  const handleGoModulePages = () => {
-    navigate(ApplicationRoutes.MODULE_ONE);
+  const lidarComIdaParaModulos = () => {
+    navigate(RotasAplicacao.MODULE_ONE);
```

Comment:
@HenriqueMamede o botão "Navegar.." parou de funcionar, ajustar a rota para "MODULO_UM"

![image](https://github.com/user-attachments/assets/0480bb29-e4e0-4539-bfa0-65b8b0a7eced)

```suggestion
    navigate(RotasAplicacao.MODULO_UM);
```